pkg/resources/cruisecontrol/configmap.go

```diff
@@ -162,30 +159,47 @@ func GenerateCapacityConfig(kafkaCluster *v1beta1.KafkaCluster, log logr.Logger,
 
 	capacityConfig := CapacityConfig{}
 
-	for _, broker := range kafkaCluster.Spec.Brokers {
-		brokerDisks, err := generateBrokerDisks(broker, kafkaCluster.Spec, log)
-		if err != nil {
-			return "", errors.WrapIfWithDetails(err, "could not generate broker disks config for broker", "brokerID", broker.Id)
+	brokerIdFromStatus := make([]string, 0, len(kafkaCluster.Status.BrokersState))
+	for key := range kafkaCluster.Status.BrokersState {
+		brokerIdFromStatus = append(brokerIdFromStatus, key)
+	}
+	sort.Strings(brokerIdFromStatus)
+	log.Info("String keys", "brokerIdFromStatus", brokerIdFromStatus)
```

Comment:
I'd make this debug log and a bit more verbose as `String keys` doesn't say to much that would help debugging.
```suggestion
	log.V(1).Info("generating broker capacity config for brokers", "brokers", brokerIdFromStatus)
```

Reply:
Oops, I forgot to remove that, it was only there for me to check if everything is okay with it.